doc/man/maestro-config.1

```diff
@@ -12,6 +12,29 @@ maestro-config \- set up or alter your Maestro configuration files
 
 .SH DESCRIPTION
 
+.PP
+The
+.I maestro config
+command should be the first command run after installing Maestro. Any Maestro deployment depends on the account information in the hidden directory created by the initial invocation of
+.I maestro config.
+
+.PP
+.I maestro config
+checks for creation of a hidden
+.I .maestro
+directory created in the users home directory and also the existence of a file
+.I aws_account_info.json
+. If the directory and file exists, and there is information present, the values present are presented to the user when prompted for account information. Otherwise, a user will simply be prompted for appropriate account information (i.e. in the case this is the initial invocation of the command).
```

Comment:
I tested this, and weirdly enough, this won't render the "if the directory and file exists..." line (27). I think line 26 needs to be `.IR aws_account_info.json .`, then line 27 can just start with text (not period) like so:
```
.IR aws_account_info.json .
If the directory and file exists...
```
.

I think the reason for this is any period at the beginning of the line will try to be interpreted as a macro. If we put the period on the line before, the issue is resolved.

Reply:
Nice catch. I'd actually started another line with a period and noticed the same error when I tested. I neglected that I'd used the same erroneous syntax on line 26.